h2o-py/tests/testdir_misc/pyunit_columns_by_type.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import print_function
+from builtins import range
+import sys
+sys.path.insert(1,"../../")
+import h2o
+from tests import pyunit_utils
+
+def lists_equal(l1,l2):
+    return len(l1) == len(l2) and sorted(l1) == sorted(l2)
```

Comment:
Isn't Rapids supposed to return the list of columns in the correct order? I think just `l1 == l2` is a better check then